src/config/redis.ts

```diff
@@ -15,5 +15,6 @@ export default {
   get: promisify(client.get).bind(client),
   set: promisify(client.set).bind(client),
   del: promisify(client.del).bind(client),
-  default: client
+  flushall: promisify(client.flushall).bind(client),
```

Comment:
flushall is not recommended in production, you can be able to generate logic if development flushall is true and production is set to false

in the real case, the production must be removed lock with a certain key

Reply:
This will only be used in testing, as we need a clear redis database for each test. However, there's no way to restrict the usage of flushall, since we can still access it using `redis.client.flushall`

Reply:
yes don't set to endpoint if for test it's okay gass